features/steps/standby_cluster.py

```diff
@@ -14,17 +14,6 @@
 callback = executable + " features/callback2.py "
 
 
-@step('I start {name:w} with callback configured')
-def start_patroni_with_callbacks(context, name):
```

Comment:
Previously we had to delay `on_role_change` callbacks until the logical slot is created on the new primary.
Behave tests were checking that callback was really executed.

Since the logical slot is already there before the promote there is no point to delay the callback and no point to test it.